draft-ietf-tls-tls13.md

```diff
@@ -1938,11 +1938,14 @@ processed and transmitted as specified by the current active session state.
                case certificate:           Certificate;
                case certificate_verify:    CertificateVerify;
                case finished:              Finished;
-               case session_ticket:        NewSessionTicket;
+               case new_session_ticket:    NewSessionTicket;
                case key_update:            KeyUpdate;
            } body;
        } Handshake;
 
+[[TODO: Add IANA Considerations to rename the 'NewSessionTicket' HandshakeType
+value to 'new_session_ticket'.]]
+
```

Comment:
Oh, I figured there might have been some very particular format or something.

I've added a sentence to the bit at the bottom that talks about HandshakeType. I see the other entries use "[SHALL update/has updated]" so I mimicked them. Does that look good?